spanner/client.go

```diff
@@ -84,7 +89,8 @@ const (
 )
 
 var (
-	validDBPattern = regexp.MustCompile("^projects/(?P<project>[^/]+)/instances/(?P<instance>[^/]+)/databases/(?P<database>[^/]+)$")
+	validDBPattern      = regexp.MustCompile("^projects/(?P<project>[^/]+)/instances/(?P<instance>[^/]+)/databases/(?P<database>[^/]+)$")
+	serverTimingPattern = regexp.MustCompile(`([a-zA-Z0-9_-]+);\s*dur=(\d+)`)
```

Comment:
The duration number could be float numbers. `dur=(\d+)` may not handle this well (I think Java has this problem as well).

Reply:
Done, added unit test TestParseServerTimingHeader